traitsui/qt4/list_editor.py

```diff
@@ -526,7 +526,8 @@ def update_editor(self):
 
             # Remember the page for later deletion processing:
             self._uis.append([ui.control, ui, view_object, monitoring])
-
+            if len(self._uis) == 1:
+                self._tab_activated(0)
```

Comment:
These fixes are a little clunky, but they get the job done.  Maybe it is worth thinking about it more and coming up with something cleaner?  

Goal here was really a quick fix